oak/server/rust/oak_runtime/src/runtime/tests.rs

```diff
@@ -15,19 +15,30 @@
 //
 
 use super::*;
+use std::sync::Once;
+
+static LOG_INIT_ONCE: Once = Once::new();
```

Comment:
Maybe add a comment saying that it is an error to set the logger more than once (I guess?)